### PR TITLE
triplet relations generated and tested.

### PR DESCRIPTION
**Pull request by Rahgooy**
triplet relations generated and tested.

### CODE REVIEW

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala [hunk 1]
```diff
@@ -115,9 +116,71 @@ object combinedPairApp extends App with Logging {
...
+    val offset = sp.getSentence.getStart
+    val lmStart = if (lm != null) offset + lm.getStart else -1
+    val lmEnd = if (lm != null) offset + lm.getEnd else -1
+    val trStart = if (tr != null) offset + tr.getStart else -1
+    val trEnd = if (tr != null) offset + tr.getEnd else -1
+    val spStart = offset + sp.getStart
+    val spEnd = offset + sp.getEnd
+    new RelationEval(trStart, trEnd, spStart, spEnd, lmStart, lmEnd)
+  }
+
```
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala:183]: We should be able to pass two properties of the data models and compare them to report P,R,F1. But for now I merge your code.

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala:183]: can you please explain more? note that we compare relations here, not roles

Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala:183]: I see. But basically, isSpatial can be a binary property of a triplet, right?

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala:183]: this is a SpRL evaluator that compares annotations for roles and relations in SpRL. For each role, we have start and end, and for each relation, we have three roles which need three pairs of inputs.
Here we gather entities from a category and count how many of them match with the actual spans and how many does not match in order to calc P, R, andF1. This is useful especially when we want to compare two Xml files and don't have access to the classifiers.

Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala:183]: This is good for now, but I think we need to revisit this evaluation issue later again.

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala:183]: ok

**Pull request merged**